code/modules/loadout/loadout_items.dm

```diff
@@ -35,6 +35,9 @@ GLOBAL_LIST_INIT(all_loadout_categories, init_loadout_categories())
 	/// Displayed name of the loadout item.
 	/// Defaults to the item's name if unset.
 	var/name
+	/// Title of a group that this item will be bundled under
+	/// Defaults to parent category's title if unset
+	var/group = null
```

Comment:
IDK if it's feasible but I'm curious... is there a world where groups could be collapsible? Hide them until a player clicks the group name?
~~I love me some collapsible lists~~

Reply:
Not sure if we want that? If anything that'd make navigating huge menus more confusing

Reply:
Yea, definitely wouldn't be as useful up here anyways, more a thought for the massive lists downstream. Kind of thing I've been debating for a few of the larger UIs that keep popping up